src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/handlers/presubmit/AppealDecidedNotificationHandler.java

```diff
@@ -0,0 +1,95 @@
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.ccd.callback.PreSubmitCallbackStage;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.handlers.PreSubmitCallbackHandler;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.service.AppealDecidedNote;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.service.HomeOfficeInstructService;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.service.NotificationsHelper;
+
+
+@Slf4j
+@Component
+public class AppealDecidedNotificationHandler implements PreSubmitCallbackHandler<AsylumCase> {
+
+    private HomeOfficeInstructService homeOfficeInstructService;
+    private NotificationsHelper notificationsHelper;
+
+    public AppealDecidedNotificationHandler(
+        HomeOfficeInstructService homeOfficeInstructService,
+        NotificationsHelper notificationsHelper
+    ) {
+        this.homeOfficeInstructService = homeOfficeInstructService;
+        this.notificationsHelper = notificationsHelper;
+    }
+
+    @Override
+    public boolean canHandle(PreSubmitCallbackStage callbackStage, Callback<AsylumCase> callback) {
+        requireNonNull(callbackStage, "callbackStage must not be null");
+        requireNonNull(callback, "callback must not be null");
+
+        return callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
+               && (callback.getEvent() == SEND_DECISION_AND_REASONS);
+    }
+
+    @Override
+    public PreSubmitCallbackResponse<AsylumCase> handle(
+        PreSubmitCallbackStage callbackStage, Callback<AsylumCase> callback) {
+
+        if (!canHandle(callbackStage, callback)) {
+            throw new IllegalStateException("Cannot handle callback");
+        }
+        log.info("Preparing to send {} notification to HomeOffice for event {}",
+            COURT_OUTCOME.toString(), callback.getEvent());
+
+        AsylumCase asylumCase = callback.getCaseDetails().getCaseData();
+
+        final String homeOfficeReferenceNumber = notificationsHelper.getHomeOfficeReference(asylumCase);
+
+        final String caseId = asylumCase.read(APPEAL_REFERENCE_NUMBER, String.class)
+            .orElseThrow(() -> new IllegalStateException("Case ID for the appeal is not present"));
+
+        final String appealDecision = asylumCase.read(IS_DECISION_ALLOWED, AppealDecision.class)
+            .orElseThrow(() -> new IllegalStateException("decision property must be set"))
+            .getValue().toUpperCase();
+
+        final AppealDecidedInstructMessage bundleInstructMessage =
+            appealDecidedInstructMessage()
+                .withConsumerReference(notificationsHelper.getConsumerReference(caseId))
+                .withHoReference(homeOfficeReferenceNumber)
+                .withMessageHeader(notificationsHelper.getMessageHeader())
+                .withMessageType(COURT_OUTCOME.toString())
```

Comment:
Try to use COURT_OUTCOME.name() as name() method is final, while the toString() may be overwritten.